.github/workflows/verify-snapshots.yml

```
@@ -133,6 +133,90 @@ jobs:
+          if-no-files-found: ignore
+
+      - name: Upload test results (attempt 2)
+        if: always() && steps.test_attempt_2.outcome != 'skipped'
+        uses: actions/upload-artifact@v4
+        with:
+          name: test-results-attempt-2
+          path: TestResults_2.xcresult
+          if-no-files-found: ignore
+
+      - name: Upload failed tests lists
+        if: always() && steps.test.attempt_1.outcome == 'failure'
```

actionlint (1.7.12)

[error] property "test" is not defined in object type {test_attempt_1: {conclusion: string; outcome: string; outputs: {string => string}}; test_attempt_2: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)